src/Core/Domain/Category/Exception/CategoryConstraintException.php

```diff
@@ -61,6 +61,11 @@ class CategoryConstraintException extends CategoryException
      */
     const TOO_MANY_MENU_THUMBNAILS = 8;
 
+    /**
+     * Code is used when invalid id is supplied.
+     */
+    const INVALID_ID = 10;
```

Comment:
1, 2, 4, 5, 6, 8.. No real order here :thinking:

Reply:
You just dont know the code :trollface:

Reply:
@PierreRambaud I noticed that the other  **xxxConstraintException** classes share the same const values, for example "INVALID_ID" is always set to 10.  I thought it would be nice to follow this existing pattern.

Reply:
Oh, weird, but ok :+1: